tools/doc/package.json

```diff
@@ -7,10 +7,11 @@
     "node": ">=6"
   },
   "dependencies": {
-    "marked": "^0.3.5",
+    "marked": "^0.3.5"
+  },
+  "devDependencies": {
     "js-yaml": "^3.5.2"
   },
-  "devDependencies": {},
```

Comment:
Why this change? `js-yaml` is a full dependency of the doctool

Reply:
At some point it you hooked `js-yaml` to redirect to the copy that is in `ESLint`.
https://github.com/nodejs/node/blob/bb4462681cd2812d718897ca7ac8b8d48bca68f0/tools/doc/node_modules/js-yaml/index.js#L1-L15
This was overwritten anytime `npm i` was run.
(I agree we should also stop running `npm i` as part of the regular flows, since `tools/doc` should work as is in git)

Reply:
Yes, not running `npm i` is the fix here. It’s too bad the plans for releasing the doctool as a standalone module didn’t get anywhere so far, but semantically this is backwards :(

Reply:
I agree. I'll roll this back as soon as I finish testing the install-les make target.